dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -337,19 +337,20 @@ private static DependencyGraph levelOrder(
   }
 
   /**
-   * Returns true if {@code exceptionAndPath.getPath} does not contain {@code optional} dependency
-   * and the path does not contain {@code scope:provided} dependency.
+   * Returns true if {@code dependencyPath} does not contain {@code optional} dependency and the
+   * path does not contain {@code scope:provided} dependency.
    */
-  private static boolean isUnacceptableResolutionException(ExceptionAndPath exceptionAndPath) {
-    ImmutableList<DependencyNode> dependencyNodes = exceptionAndPath.getPath();
+  public static boolean isUnacceptableResolutionExceptionAt(List<DependencyNode> dependencyPath) {
```

Comment:
I can't parse this name. Can you think of a more descriptive one?

Reply:
Renamed this to `unacceptableMissingDependency`.